packages/dashmate/configs/defaults/getTestnetConfigFactory.js

```diff
@@ -25,7 +25,7 @@ export default function getTestnetConfigFactory(homeDir, getBaseConfig) {
       },
       core: {
         docker: {
-          image: 'dashpay/dashd:20.1.0-nightly.2024.02.15',
+          image: 'dashpay/dashd:20.1.0',
```

Comment:
Maybe `20.1`, so masternodes can issue `dashmate update` and receive minor upgrades? Or even `20`

Reply:
I believe want to control what specific version of Core running on testnet. @PastaPastaPasta ?

Reply:
Not sure all the implications of this version here. Probably fine to be 20.1